### DISCUSSION TITLE
version 2.1.1 issues

### DISCUSSION THREAD

**Discussion by AndyIreland** [cedricp/ddt4all#866] [Q&A]
dont like this version 2.1.1 (32bit)

interface does not fill full screen and I cannot drag the sidebar (arrowed in red) with my mouse any more. 

![image](https://github.com/cedricp/ddt4all/assets/49172217/3aa6816d-878a-46c7-8f10-5506b87e89d7)

have gone back to older version release Nov 2020 python3 version or 'all cars' from Drive2 - much better

**Comment by Furtif** [comment]
> dont like this version 2.1.1 (32bit)
> 
> interface does not fill full screen and I cannot drag the sidebar (arrowed in red) with my mouse any more.
> 
> ![image](https://user-images.githubusercontent.com/49172217/249907575-3aa6816d-878a-46c7-8f10-5506b87e89d7.png)
> 
> have gone back to older version release Nov 2020 python3 version or 'all cars' from Drive2 - much better

maybe use old database version... new cars in 2.1.1 are made according to recent database version.
can use the ecu.zip given here : https://github.com/cedricp/ddt4all/discussions/845
